docs/docs/05-migrations/13-v8-to-v9.md

```diff
@@ -210,9 +210,20 @@ The `IterateConsensusMetadata` function has been removed.
 
 ### 08-wasm
 
+- The `Initialize`, `Status`, `GetTimestampAtHeight`, `GetLatestHeight`, `VerifyMembership`, `VerifyNonMembership`, `VerifyClientMessage`, `UpdateState` and `UpdateStateOnMisbehaviour` functions in `ClientState` have been removed and all their logic has been moved to functions of the `LightClientModule`.
+- The `MigrateContract()` function has been removed from `ClientState`.
```

Comment:
nit: I don't feel strongly about using `()` when we refer to functions, but for consistency with the rest of the document we should either remove them or add them to all.

Reply:
Fine with removing the parentheses  👍🏻